README.md

```diff
@@ -8,26 +8,74 @@
 [![LoC](https://tokei.rs/b1/github/cosmos/gaia)](https://github.com/cosmos/gaia)
 [![GolangCI](https://golangci.com/badges/github.com/cosmos/gaia.svg)](https://golangci.com/r/github.com/cosmos/gaia)
 
-This repository hosts `Gaia`, the first implementation of the Cosmos Hub based on the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
 
-**Note**: Requires [Go 1.14+](https://golang.org/dl/)
+## Mainnet Full Node Quick Start
 
-## Cosmos Hub Mainnet
+This assumes that you're running Linux or MacOS and have installed [Go 1.14+](https://golang.org/dl/).  This guide helps you:
 
-To run a full-node for the mainnet of the Cosmos Hub, first [install `gaiad`](./docs/gaia-tutorials/installation.md), then follow [the guide](./docs/gaia-tutorials/join-mainnet.md).
+* build and install Gaia
+* allow you to name your node
+* add seeds to your config file
+* download genesis state
+* start your node 
+* use gaiacli to check the status of your node.  
 
-For status updates and genesis file, see the [launch repo](https://github.com/cosmos/launch).
+Build, Install, and Name your Node
+```bash
+# Clone Gaia
+git clone -b v2.0.9 https://github.com/cosmos/gaia
+# Enter the folder Gaia was cloned into
+cd gaia
+# Comile and install Gaia
+make install
+# Initalize Gaiad in ~/.gaiad and name your node
+gaiad init yournodenamehere
+```
 
-## Quick Start
+Add Seeds
+```bash
+# Edit config.toml
+nano ~/.gaiad/config/config.toml
+```
 
+Scroll down to seeds in `config.toml`, and add some of these seeds as a comma-separated list:
 ```
-make install
+ba3bacc714817218562f743178228f23678b2873@5.83.160.108:26656
+1e63e84945837b026f596ed8ae68708783d04ad4@51.75.145.123:26656
+d2d452e7c9c43fa5ef017552688de60a5c0053ee@34.245.217.163:26656
+dd36969b56c740bb40bb8badd4d4c6facc35dc24@206.189.115.41
+a0aca8fb801c69653a290bd44872e8457f8b0982@47.99.180.54
+27f8dd3bdbecbef7192291083706c156e523d8e0@3.122.248.21
+aee0df1a660f301d456a0c2f805b372f7341e8ec@63.35.230.143:26656
+7d1f660b361d6286715c098a3a171e554e9642bb@34.254.205.37
+fa105c2291ac4aa452552fa4835266300a8209e1@88.198.41.62
+bd410d4564f7e0dd9a0eb16a64c337a059e11b80@47.103.35.130
+```
+
+Download Genesis, Start your Node, Check your Node Status
+```bash
+# Download genesis.json
+curl https://raw.githubusercontent.com/cosmos/launch/master/genesis.json > $HOME/.gaiad/config/genesis.json
```

Comment:
`wget https://raw.githubusercontent.com/cosmos/launch/master/genesis.json` should be enough (and in some bare metail images `curl` may not even be available). But it's not a big deal at all.

Reply:
you're right :).  

I'll make this a wget situation.